packages/app-store/googlevideo/api/add.ts

```diff
@@ -0,0 +1,46 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import prisma from "@calcom/prisma";
+
+import getInstalledAppPath from "../../_utils/getInstalledAppPath";
+
+/**
+ * This is an example endpoint for an app, these will run under `/api/integrations/[...args]`
+ * @param req
+ * @param res
+ */
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
Need to use the standard approach of adding API endpoint here. See sendgrid/api/add.ts for example.